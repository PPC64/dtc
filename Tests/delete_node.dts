/dts-v1/;

/ {
    foo {
    };

    bar {
    };
};

/ {
    /delete-node/ bar;
};

