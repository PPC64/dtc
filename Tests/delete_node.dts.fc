/dts-v1/;

/ {
    foo {
    };

    bar {
    };
 // CHECK: Error: Attempts to delete node that was not
 // CHECK: added in merge: bar, while checking node: /
 // [-Wdeleted-nodes]
    /delete-node/ bar;
};
