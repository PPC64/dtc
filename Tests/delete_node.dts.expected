/dts-v1/;

/  {

	foo {

	};
};
